### PR TITLE
Added support for FS quotas on extra mountpoints

### PR DESCRIPTION
**Pull request by jbahillo**
Just for Jenkins test

### CODE REVIEW

**Comment by jbahillo** [comment]
Jenkins retest this please

File: main/samba/src/scripts/user-quota [hunk 1]
```diff
@@ -30,8 +30,9 @@
 # uidNumber: user uidNumber
 # quota: quota size in KB
 use Quota;
-
-sub currentQuota
```
Comment (papajulio) [inline main/samba/src/scripts/user-quota]: This shouldn't be deleted right?

Comment (papajulio) [inline main/samba/src/scripts/user-quota]: By the way in this method, we should do the same and check the other new mount_points to return the actual quota right?

**Comment by jbahillo** [comment]
Right about the first, not (or not too relevant) about the second.

Our support will consist on extending the SAME quota applied to /home as to the rest of FileSystems. Given this quota should be cosnsistent on every FS, and it should not matter which FS we check, the rest should and will have the same quota applied

**Comment by jbahillo** [comment]
Jenkins retest this please

**Comment by papajulio** [comment]
:+1:

**Pull request merged**